tests/manage/z_cluster/test_scc.py

```diff
@@ -0,0 +1,226 @@
+                "downwardAPI",
+                "emptyDir",
+                "persistentVolumeClaim",
+            ],
+        }
+
+        # Add new scc to system:authenticated
+        OCP().exec_oc_cmd(
+            command=f"adm policy add-scc-to-group {scc_factory(scc_dict=scc_dict).name} system:authenticated"
+        )
+
+        # Delete csi-provisioner and noobaa db pods
+        labels = [
+            constants.NOOBAA_DB_LABEL_47_AND_ABOVE,
+            constants.CSI_RBDPLUGIN_PROVISIONER_LABEL,
+            constants.CSI_CEPHFSPLUGIN_PROVISIONER_LABEL,
+        ]
+        pods = list()
+        for label in labels:
+            pods.extend(
+                get_pods_having_label(
+                    label=label, namespace=constants.OPENSHIFT_STORAGE_NAMESPACE
+                )
+            )
+        pods = [Pod(**pod) for pod in pods]
+        for pod in pods:
+            pod.delete()
+
+        # wait for the pods to reconcile
+        wait_for_pods_to_be_running(pod_names=[pod.name for pod in pods])
+        logger.info("Pods reconciled successfully without any issues!!")
+
+        # cluster sanity health check
+        try:
+            Sanity().health_check()
+        except Exception as ex:
+            logger.error("Failed at cluster health check!!")
+            raise ex
+
+    @pytest.fixture()
+    def setup(self, project_factory, pvc_factory, teardown_factory):
+
+        # create a project for simple-app deployment
+        project = project_factory(project_name="test-project")
+
+        # create pvc
+        pvc = pvc_factory(
+            project=project,
+            access_mode=constants.ACCESS_MODE_RWO,
+            size=20,
+        )
+        logger.info(f"Pvc created: {pvc.name}")
+
+        # create service account
+        service_account_obj = helpers.create_serviceaccount(project.namespace)
+        helpers.add_scc_policy(service_account_obj.name, project.namespace)
+        teardown_factory(service_account_obj)
+
+        # create simple-app deployment
+        simple_app_data = templating.load_yaml(constants.SIMPLE_APP_POD_YAML)
+        simple_app_data["metadata"]["namespace"] = project.namespace
+        simple_app_data["spec"]["template"]["spec"][
+            "serviceAccountName"
+        ] = service_account_obj.name
+        simple_app_data["spec"]["template"]["spec"]["volumes"][0][
+            "persistentVolumeClaim"
+        ]["claimName"] = pvc.name
+        simple_app_dc = helpers.create_resource(**simple_app_data)
+        teardown_factory(simple_app_dc)
+
+        simple_app_dc_obj = get_deployments_having_label(
+            label="app=simple-app", namespace=project.namespace
+        )[0]
+        simple_app_pod = Pod(
+            **get_pods_having_label(
+                label="app=simple-app", namespace=project.namespace
+            )[0]
+        )
+        helpers.wait_for_resource_state(
+            resource=simple_app_pod, state=constants.STATUS_RUNNING, timeout=300
+        )
```

Comment:
This is an app pod YAML with fsGroupChangePolicy "OnRootMismatch", maybe we can convert this to a fixture so that it can be used in other tests if needed.

Reply:
done accomodated this option in pod_factory() fixture